src/test/js/multibranch/filtering.js

```diff
@@ -0,0 +1,55 @@
+const jobName = 'withGitFlow';
+const path = require("path");
+const pathToRepo = path.resolve('./target/testfilter-project-folder');
+const soureRep = './src/test/resources/multibranch_2';
+const git = require("../../../main/js/api/git");
+
+/** @module filtering
+ * @memberof multibranch
+ * @description Check that can filter the activity list
+ */
+module.exports = {
+
+
+    // ** creating a git repo */
+    before: function (browser, done) {
+          // we creating a git repo in target based on the src repo (see above)
+          git.createRepo(soureRep, pathToRepo)
+              .then(function () {
+                  git.createBranch('feature/1', pathToRepo)
+                      .then(done);
```

Comment:
Tells nightwatch that the test is "done" when async shit is happening, right? I don't see the full diff and I'm too lazy to open it :)